projects/schematics/src/shared/utils/graph-utils_spec.ts

```diff
@@ -167,7 +169,7 @@ describe('Graph utils', () => {
 
   describe('feature dependency graph', () => {
     it('should generate the correct installation order', () => {
-      expect(crossFeatureInstallationOrder).toMatchInlineSnapshot(`
+      expect(crossFeatureInstallationOrder).toMatchSnapshot(`
```

Comment:
`minor:` I'd rather revert this change and manually modify the inline snapshot to ensure the test works as expected for the previous state before changes.

Reply:
but `toMatchInlineSnapshot` doesn't work?